.coderrabbit.yml

```diff
@@ -0,0 +1,22 @@
+language: "en-US"
+reviews:
```

Comment:
make the reviews profile to be assertive. I think the nit picks by @coderabbitai are actually good.